### PR TITLE
Move VS integration tests to Azure Dev Ops

### PR DESCRIPTION
**Pull request by jaredpar**

### CODE REVIEW

File: build/scripts/build.ps1 [hunk 1]
```diff
@@ -102,6 +102,7 @@ function Process-Arguments() {
     if ($cibuild -and -not $official -and $anyVsi) {
         # Avoid spending time in analyzers when requested, and also in the slowest integration test builds
         $script:skipAnalyzers = $true
+        $script:bootstrap = $false
```
Comment (sharwell) [inline build/scripts/build.ps1]: 💭 I wouldn't complain about a corresponding update to the preceding comment

File: build/scripts/build.ps1 [hunk 2]
```diff
@@ -394,7 +395,7 @@ function Test-XUnit() {
 
     if ($cibuild) {
         # Use a 75 minute timeout on CI
```
Comment (xPaw) [inline build/scripts/build.ps1]: comment wasn't updated

File: build/scripts/build.ps1 (hunk 1 above)
Comment (jaredpar) [reply build/scripts/build.ps1]: I have a followup PR for this. Will update the comment. 

Using a follow up PR cause I want to get this merged and get us out of the intermmediate state we're in ATM.

File: build/scripts/build.ps1 (hunk 2 above)
Comment (jaredpar) [reply build/scripts/build.ps1]: Have a followup PR coming. Will fix.

**Pull request merged**